apps/docs/src/app/core/component-docs/select/select-docs.component.html

```diff
@@ -1,107 +1,130 @@
+    <fd-select-nested-options></fd-select-nested-options>
 </component-example>
-<code-example [exampleFiles]="selectViewValue"></code-example>
+<code-example [exampleFiles]="selectExtendedOptions"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'selectMaxHeight'" [componentName]="'select'">
-    Select Max Height
+<fd-docs-section-title [id]="'custom-view-value'" [componentName]="'select'">
+    Custom View Value
 </fd-docs-section-title>
 <description>
-    The select component allows to define <code>[maxHeight]</code>. It can be useful when there are a lot of options,
-    the select body is be scrollable. By default maximum height of select body is 45% of browsers window inner height.
+    The value which appears inside the Select control can be easily customized using the OptionComponent
+    <code>[viewValue]</code>
+    input.
 </description>
 <component-example>
-    <fd-select-max-height-example></fd-select-max-height-example>
+    <fd-select-view-value-example></fd-select-view-value-example>
 </component-example>
-<code-example [exampleFiles]="selectMaxHeight"></code-example>
+<code-example [exampleFiles]="selectViewValue"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'nestedOptions'" [componentName]="'select'">
-    Nested Options
+<fd-docs-section-title [id]="'select-mobile'" [componentName]="'select'">
+    Select In Mobile Mode
 </fd-docs-section-title>
 <description>
-    Instances of the fd-option component will be found regardless of how deep in the hierarchy they are. It also
-    includes having options inside options, for instance. This feature allows for some very flexible usage. The example
-    below showcases how to add a scrolling container around the options. Note that accessibility generally requires some
-    text-content to be present in the actual option element.
+    To open a Menu in mobile mode, compose the Menu Component together with the Dialog Component.<br/>
+    There are a few important things to remember:
+    <ul>
+        <li>
+            Because Select is tightly bound to Option, it is important to keep the options in
+            view without instantiating and destroying them with every dialog open/close action. We can easily achieve
+            this by creating the Dialog once and using the <code>DialogRef.hidden()</code> method to show/hide the dialog.
+        </li>
+
+        <li>
+            To get the full Fiori3 experience, set Select in mobile mode using <code>[mobile]="true"</code>. Also remember to instantiate the
+            Dialog inside Select using <code>DialogConfig.container</code> and <code>Select.dialogContainerElementRef</code>.
+        </li>
+
+        <li>
+            The Options template needs to be declared inside the <code>fd-select</code> tags to properly configure element
+            injectors. Use <code>Select.selectOptionsListTemplate</code> to render options in the desired place.
+        </li>
```

Comment:
Should we add breaks between the bullet points to improve readability?

<img width="954" alt="Screen Shot 2020-04-27 at 9 29 34 AM" src="https://user-images.githubusercontent.com/54723167/80378390-9707c080-886a-11ea-8f0e-fb47772ea9ac.png">